app/code/Magento/NewRelicReporting/Test/Mftf/Test/AdminChecksSystemConfigDependencyTest.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<tests xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/testSchema.xsd">
+    <test name="AdminChecksSystemConfigDependencyTest">
+        <annotations>
+            <features value="NewRelicReporting"/>
+            <stories value="Admin is able to see the configuration fields only after enabling the feature"/>
+            <title value="Admin can see the configuration fields only after enabling the feature"/>
+            <description value="The system configs should be available only after enabling the New Relic feature."/>
+            <severity value="CRITICAL"/>
+            <group value="NewRelicReporting"/>
+        </annotations>
+        <before>
+            <actionGroup ref="LoginAsAdmin" stepKey="loginAsAdmin"/>
+            <actionGroup ref="AdminNavigateToNewRelicConfigurationActionGroup" stepKey="goToConfigPage"/>
+            <actionGroup ref="AdminExpandNewRelicConfigTabActionGroup" stepKey="expandingGeneralTab">
+                <argument name="tabName" value="general"/>
+            </actionGroup>
+        </before>
+        <after>
+            <actionGroup ref="logout" stepKey="logoutOfAdmin"/>
+        </after>
+        <actionGroup ref="AssertAdminNewRelicConfigFieldIsNotVisibleActionGroup" stepKey="checkingIfApiUrlIsNotVisible">
+            <argument name="config" value="{{AdminNewRelicConfigSystemSection.apiUrl}}"/>
+        </actionGroup>
+        <actionGroup ref="AdminUncheckNewRelicUseSystemValueActionGroup" stepKey="uncheckUseSystemValue"/>
```

Comment:
I do suggest to create generic action group in `Magento_Config` module which will uncheck `Use system value` by passing argument which one should be unchecked.
I think you can pass to such action group selector to config row. Your selector should look similar to
`#{{configRowId}} > .use-default > input` or maybe you will find any other better selector to approach this checkbox.

Reply:
Implemented. The only thing that I wasn't able to do, is to store the `configRowId` in Section, and  pass it as argument, instead of hardcoding the argument value.
If you have any suggestions here, please advice.